docs/ci.rst

```diff
@@ -1,6 +0,0 @@
-.. _ci:
-
-Continuous Integration
-======================
-
-Needs writing!
```

Comment:
Do we still not want something on the CI?

Reply:
Probably want to document our specifics yes.

Reply:
Probably open an issue so we don't forget.

Reply:
#2

Reply:
Thanks.